### ISSUE TITLE
[Outreachy Task Submission] Accessibility Issue with Keyboard Navigation in Survey Settings

### ISSUE THREAD

**Issue by Chiemezuo** [ushahidi/platform#4795]
# Intro
In the **Surveys** part of the 'Settings' page, keyboard navigation doesn't seem to support the `enter` button. This makes it impossible for alternative forms of site navigation from a mouse. This is a violation of WCAG Success Criterion 2.1.1 (Level A) - keyboard. [See](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html)

## Description
Success Criterion 2.1.1 (Level A) - keyboard requires that all functionality of the content is operable through a keyboard interface without requiring specific timings for individual keystrokes, except where the underlying function requires input that depends on the path of the user's movement and not just the endpoints.

## Impact
The inability to fully navigate with a keyboard would affect users with visual impairments, screen readers, or alternative forms of navigation (other than a mouse). They would be unable to use our platform.

## Fix
I will need to configure a `keydown.enter` attribute for the survey list items.
